src/intro_gui.cpp

```diff
@@ -333,17 +371,14 @@ struct SelectGameWindow : public Window {
 
 		if (str != 0) {
 			int height = GetStringHeight(str, size->width);
-			if (height > 3 * FONT_HEIGHT_NORMAL) {
-				/* Don't let the window become too high. */
-				Dimension textdim = GetStringBoundingBox(str);
-				textdim.height *= 3;
-				textdim.width -= textdim.width / 2;
-				*size = maxdim(*size, textdim);
-			} else {
+			/* Don't let the window become too high - allow up to 7 lines. */
+			while (height > 7 * FONT_HEIGHT_NORMAL) {
```

Comment:
Could use some constants to replace magic numbers. Or at least an explanation of the `size->width * 7 / 5`: where does the 5 come from?